src/github.ts

```diff
@@ -628,27 +635,28 @@ export class GitHub {
     logger.debug(`Fetching releases with cursor ${cursor}`);
     const response = await this.graphqlRequest({
       query: `query releases($owner: String!, $repo: String!, $num: Int!, $cursor: String) {
-        repository(owner: $owner, name: $repo) {
-          releases(first: $num, after: $cursor, orderBy: {field: CREATED_AT, direction: DESC}) {
-            nodes {
-              name
-              tag {
-                name
-              }
-              tagCommit {
-                oid
+            repository(owner: $owner, name: $repo) {
+              releases(first: $num, after: $cursor, orderBy: {field: CREATED_AT, direction: DESC}) {
+                nodes {
+                  name
+                  tag {
+                    name
+                  }
+                  tagCommit {
+                    oid
+                  }
+                  url
+                  description
+                  isDraft
+                  isPrerelease
```

Comment:
Is this the only addition to the graphql that you're asking for `isPrerelease`, are the rest of the updates reformating?

Reply:
I meant to make it consistent with having the draft in the query. I'm gonna better remove it

Reply:
yes. I'm gonna revert the reformating.